package.json

```diff
@@ -35,7 +40,6 @@
     "watch": "jest --watch"
   },
   "dependencies": {
-    "@vertexvis/utils": "^0.9.8",
```

Comment:
JC, did you run into an issue with this package?

Reply:
I was trying to use this with Deno and the utils lib is using Node's crypto, which was throwing an error. The lib won't work with Deno without other changes, so it's not a big deal, but it seemed overkill to include this lib for a one-liner.